server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTest.java

```diff
@@ -66,6 +68,37 @@
     mutualSslClient = TestClients.mutualSslClient();
   }
 
+  //---------------------------------------------------------------------------------------
+  // renameSecret
+  //---------------------------------------------------------------------------------------
+
+  @Test public void renamesSecret() throws Exception {
```

Comment:
nit: test name is pretty similar to the helper method`renameSecret`, maybe `successfullyRenamesSecret` instead?